src/components/DsfrFileUpload/DsfrFileUpload.vue

```diff
@@ -41,6 +41,11 @@ const onChange = ($event: InputEvent) => {
       :for="id"
     >
       {{ label }}
+      <span
+        v-if="$attrs.required"
```

Comment:
si on met ici `'required' in $attrs && $attrs.required !== false`, cela permet au développeur de mettre simplement
`required` sur le composants `DsfrFileUpload` au lieu de `:required="true"`.
C’est ce que j’ai fait sur `DsfrInput`